### PR TITLE
gh-124228: Fix UUID configuration in configure files for NetBSD compatibility

### PR DESCRIPTION
**Pull request by furkanonder**
* Issue: gh-124228

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -3802,6 +3802,11 @@ AS_VAR_IF([have_uuid], [missing], [
   ])
 ])
 
+if test "$ac_sys_system" = "NetBSD"; then
```
Comment (vstinner) [inline configure.ac:3807]: Please add a comment with a reference to `gh-124228` explaining that libuuid is available but doesn't fit Python requirements. I understand that NetBSD libuuid only supports UUID4.

**Review by vstinner — APPROVED** [review]
LGTM

**Comment by vstinner** [comment]
Merged, thanks.

**Comment by furkanonder** [comment]
> Merged, thanks.

You are welcome.

**Pull request merged**